### PR TITLE
Add notes to 01Oct Townhall

### PR DESCRIPTION
**Pull request by gjsyme**
Notes taken during the call

### CODE REVIEW

**Comment by kempsterrrr** [comment]
Hero @gjsyme 🙏🏻 +1000 for doing these, thanks. Had a quick skim an looks good to me we're just missing action items and link to recording. Will jump on when I'm back homein an hour or two and have a propper look/suggests those additions.

**Comment by gjsyme** [comment]
I didn't catch any action items, but added the audio recording to the repo and set links to it and to youtube.

File: town-hall/2021-10-01.md [hunk 1]
```diff
@@ -141,4 +141,6 @@ D_D should boost up member content via share-your-content
 
 ### Actions Items
```
Comment (kempsterrrr) [inline town-hall/2021-10-01.md:143]:
```suggestion
- anyone interested in pariticapting in proposals and longer-form discussion related to DAO set up discourse profile @ [forum.developerdao.com](https://forum.developerdao.com/)
- `with-heart` to oragnise mob progamming session as intro to chakra-ui.
- `marc` to set up mirror profile and share with community to kick off pitch for the $WRITE race
- `kempsterrrr` too gather community availability to re-schedule more inclusive slot(s) for comunity call
```

**Review by kempsterrrr — COMMENTED** [review]
Think that's all the action items :)

**Pull request merged**